tests/contrib/flask/test_flask_appsec.py

```diff
@@ -37,7 +34,33 @@ def dynamic_url(item):
         assert resp.status_code == 200
         spans = self.pop_spans()
         root_span = spans[0]
-        assert dict(_context.get_item("http.request.path_params", span=root_span)) == {"item": "attack"}
+
+        flask_args = root_span.get_tag("flask.view_args.item")
+        path_params = _context.get_item("http.request.path_params", span=root_span)
+
+        assert path_params == {"item": "attack"}
+        assert flask_args == "attack"
```

Comment:
nit: This assert would be better off right after the variable assign, as it is not used from line 38 onwards